plugins/modules/mso_schema_template_anp_epg_contract.py

```diff
@@ -36,31 +38,68 @@
     required: true
   epg:
     description:
-    - The name of the EPG to manage.
+    - The name of the EPG.
     type: str
     required: true
+  force_replace:
+    description:
+    - Replaces all the configured contract(s) with the provided contract(s).
+    - This option can only be used in combination with the O(contracts) option.
+    - In combination with the O(state=absent) and without any contract configuration all confirgured static port(s) will be removed.
+    type: bool
   contract:
     description:
-    - A contract associated to this EPG.
+    - The Contract associated to this EPG.
+    - This option can not be used in combination with the I(contracts) option.
     type: dict
     suboptions:
       name:
         description:
-        - The name of the Contract to associate with.
+        - The name of the Contract.
+        required: true
+        type: str
+      schema:
+        description:
+        - The name of the Schema that defines the referenced Contract.
+        - If this parameter is unspecified, it defaults to the current schema.
+        type: str
+      template:
+        description:
+        - The name of the Template that defines the referenced Contract.
+        - If this parameter is unspecified, it defaults to the current schema.
+        type: str
+      type:
+        description:
+        - The type of the Contract.
+        type: str
+        required: true
+        choices: [ consumer, provider ]
+  contracts:
+    description:
+    - A list of Contracts associated to this EPG.
+    - This option can not be used in combination with the O(contract) option.
+    - All configured contract(s) will be replaced with the provided contract(s) when used with O(force_replace=true).
```

Comment:
```suggestion
    - All configured contract(s) will be replaced with the provided contract(s) when used with O(force_replace=true).
    - Only the provided contract(s) will be added, updated or removed when used with O(force_replace=false).
```

Reply:
Also added another line to explain O(state=query) behaviour